### PR TITLE
Move barcode scanning detection to barcode plugin

### PR DESCRIPTION
**Pull request by kitsuta**
Requires https://github.com/rams/barcode/pull/21.

### CODE REVIEW

File: panels/templates/attractions_admin/checkin.html [hunk 1]
```diff
@@ -93,36 +93,11 @@
...
-            } else if(response['badge_num']) {
-              var badgeNum = response['badge_num'];
-              $badgeNum.val(badgeNum);
-              getSignups(badgeNum);
-            }
-          }
-        });
-      }
-    });
+      {% set detect_barcode_scan = True %}
```
Comment (RobRuana) [inline panels/templates/attractions_admin/checkin.html:96]: Does this actually work? Wouldn't all of the code in `baseextra.html` already be rendered by the time we reach this line (meaning `detect_barcode_scan` would be `False` at the time the barcode plugin `{% extends "baseextra.html" %}`)?

I'm actually not sure! I'm going to pull this branch locally and test it right now!

Comment (kitsuta) [reply panels/templates/attractions_admin/checkin.html:96]: Oh shoot, that's a very good point. I'll try to brainstorm another tactic. Let me know the testing results!

Comment (kitsuta) [reply panels/templates/attractions_admin/checkin.html:96]: Did you ever get this tested?

Comment (RobRuana) [reply panels/templates/attractions_admin/checkin.html:96]: Oh, no! Fell off my radar ☹️

**Pull request merged**